molgenis-data-elasticsearch/src/main/java/org/molgenis/data/elasticsearch/ElasticsearchRepositoryCollection.java

```diff
@@ -92,11 +92,12 @@ public void deleteEntityMeta(String entityName)
 	@Override
 	public void addAttribute(String entityName, AttributeMetaData attribute)
 	{
-		EntityMetaData entityMetaData = dataService.getEntityMetaData(entityName);
+		EntityMetaData entityMetaData = dataService.getMeta().getEntityMetaData(entityName);
```

Comment:
The above change, I understand, but this one I find a little worrisome. Does it mean that after annotating the dataService thinks the repo has different metadata than the metadataservice thinks it has?